HW9.md

```diff
@@ -0,0 +1,181 @@
+    inet6 fe80::dfdb:481:367b:712/64 scope link noprefixroute
+       valid_lft forever preferred_lft forever
+```
+
+   #### 1.2 using centos network configuration file (statefull)
+
+```bash
+[hero@centos ~]$ sudo nano /etc/sysconfig/network-scripts/ifcfg-ens33:1
+TYPE="Ethernet"
+PROXY_METHOD="none"
+BROWSER_ONLY="no"
+BOOTPROTO="none"
+DEFROUTE="yes"
+IPV4_FAILURE_FATAL="no"
+IPV6INIT="yes"
+IPV6_AUTOCONF="yes"
+IPV6_DEFROUTE="yes"
+IPV6_FAILURE_FATAL="no"
+IPV6_ADDR_GEN_MODE="stable-privacy"
+IPADDR="192.168.225.129"
+DEVICE="ens33:1"
+ONBOOT="yes"
+[hero@centos ~]$ systemctl restart network
+==== AUTHENTICATING FOR org.freedesktop.systemd1.manage-units ===
+Authentication is required to manage system services or units.
+Authenticating as: hero
+Password:
+==== AUTHENTICATION COMPLETE ===
+[hero@centos ~]$ ip a
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+    inet 127.0.0.1/8 scope host lo
+       valid_lft forever preferred_lft forever
+    inet6 ::1/128 scope host
+       valid_lft forever preferred_lft forever
+2: ens33: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
+    link/ether 00:0c:29:dd:4d:e3 brd ff:ff:ff:ff:ff:ff
+    inet 192.168.225.128/24 brd 192.168.225.255 scope global noprefixroute dynamic ens33
+       valid_lft 1799sec preferred_lft 1799sec
+    inet 192.168.225.129/24 brd 192.168.225.255 scope global secondary noprefixroute ens33:1
+       valid_lft forever preferred_lft forever
+    inet6 fe80::dfdb:481:367b:712/64 scope link tentative noprefixroute
+       valid_lft forever preferred_lft forever
+[hero@centos ~]$ sudo nano /etc/sysconfig/network-scripts/ifcfg-ens33:1 #removed config
+[hero@centos ~]$ systemctl restart network
+==== AUTHENTICATING FOR org.freedesktop.systemd1.manage-units ===
+Authentication is required to manage system services or units.
+Authenticating as: hero
+Password:
+==== AUTHENTICATION COMPLETE ===
+[hero@centos ~]$ ip a
+1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+    inet 127.0.0.1/8 scope host lo
+       valid_lft forever preferred_lft forever
+    inet6 ::1/128 scope host
+       valid_lft forever preferred_lft forever
+2: ens33: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
+    link/ether 00:0c:29:dd:4d:e3 brd ff:ff:ff:ff:ff:ff
+    inet 192.168.225.128/24 brd 192.168.225.255 scope global noprefixroute dynamic ens33
+       valid_lft 1799sec preferred_lft 1799sec
+    inet6 fe80::dfdb:481:367b:712/64 scope link noprefixroute
+       valid_lft forever preferred_lft forever
+```
+
+   #### 1.3 using nmcli utility (statefull)
+
+```bash
+[hero@centos ~]$ sudo nmcli con mod ens33 +ipv4.addresses "192.168.225.129/24"
+[hero@centos ~]$ sudo reboot
```

Comment:
Do we have to reboot or there is another way to reload the configuration?

Reply:
`sudo systemctl restart networkmanager`
or
```
sudo nmcli networking off
sudo nmcli networking on
```